bin/moodle-docker-compose.cmd

```diff
@@ -35,8 +35,27 @@ if exist %filename% (
 )
 
 IF NOT "%MOODLE_DOCKER_BROWSER%"=="" (
```

Comment:
We should drop this condition since now the code looks for `MOODLE_DOCKER_BROWSER_NAME` which is defined inside this loop